### PR TITLE
feat(payment): errors

### PR DESCRIPTION
**Pull request by daniloab**

### CODE REVIEW

File: docs/payment/payment-failed-errors.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+tags:
+  - payment
+  - error
+---
+
+## Erros e Descrições
+
+| Código do Erro     | Descrição                                                 | Detalhe  |
+|--------------------|-----------------------------------------------------------|----------|
+| ALIAS_LIMIT_ERROR  | Limite de consultas chave pix excedido                    |  Toda conta possui um limite de consulta de chaves pix sem ter o pagamento liquidado. |
```
Comment (sibelius) [inline docs/payment/payment-failed-errors.md:13]: What is the limit?

Comment (sibelius) [reply docs/payment/payment-failed-errors.md:13]: Where can I read more about those limits?

File: docs/payment/payment-failed-errors.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+| PAY_PIX_KEY_ERROR            | Erro ao realizar o pagamento para a chave pix informada   | Erro ao processar o pagamento para a chave pix do pagamento |
+| PIX_KEY_INFO_ERROR | Erro ao consultar informações da chave pix do pagamento   | Erro durante a consulta de informações da chave pix |
+| PIX_KEY_INFO_NOT_FOUND         | Chave pix informada não encontrada/registrada em um banco | A chave pix não esta registrada em uma instituição bancária |
+| REACH_TRANSFER_LIMIT    | Limite de trasferência atingido                           | A conta atingiu o limite diurno/noturno de transaferência pix |
```
Comment (sibelius) [inline docs/payment/payment-failed-errors.md:17]: Transferência, typo

File: docs/webhook/examples/payment-payload.mdx [hunk 3]
```diff
@@ -40,6 +40,11 @@ Os webhooks de `Pagamento Externo com Falha (Pix Out)` enviam objetos com os seg
 
 - `payment`: Objeto com detalhes do Pagamento
 - `transaction`: Objeto com detalhes da Transação Pix se ela existir
+- `error`:  Objeto com o código e descrição do motivo da falha.
```
Comment (sibelius) [inline docs/webhook/examples/payment-payload.mdx:43]: Can you describe the object?

File: docs/payment/payment-failed-errors.md (hunk 2 above)
Comment (daniloab) [reply docs/payment/payment-failed-errors.md:17]: tks

File: docs/webhook/examples/payment-payload.mdx (hunk 3 above)
Comment (daniloab) [reply docs/webhook/examples/payment-payload.mdx:43]: it is describe in the example linked with the documentation on info section

**Pull request merged**